src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,181 @@
+    minFrequency: OptionalAttribute(0x05, TlvUInt16, {default: 0}),
+
+    /** Max value of CurrentFrequency capable of being assigned. */
+    maxFrequency: OptionalAttribute(0x06, TlvUInt16, {default: 0}),
+
+    /** Time to move to/from the target when On/Off received by On/Off cluster on the same endpoint. */
+    onOffTransitionTime: OptionalWritableAttribute(0x10, TlvUInt16, {default: 0}),
+
+    /** CurrentLevel when OnOff attr of On/Off cluster on the same endpoint is TRUE. */
+    onLevel: WritableAttribute(0x11, TlvNullable(TlvUInt8), {default: null}),
+
+    /** Time taken to move the current level from the min level to the max level. */
+    onTransitionTime: OptionalWritableAttribute(0x12, TlvNullable(TlvUInt16,), {default: null}),
+
+    /** Time taken to move the current level from the max level to the minlevel. */
+    offTransitionTime: OptionalWritableAttribute(0x13, TlvNullable(TlvUInt16), {default: null}),
+
+    /** Default rate in units per second when a Move command has a null rate. */
+    defaultMoveRate: OptionalWritableAttribute(0x14, TlvNullable(TlvUInt8)),
+
+    /** Determines the default behavior of some cluster commands. */
+    options: Attribute(0x0f, optionsBitmap),
+
+    /** Desired startup level for a device when it is supplied with power. */
+    startUpCurrentLevel: OptionalWritableAttribute(0x4000, TlvNullable(TlvUInt8)),  
+};
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.6 */
+const commands = {
+    
+    /** Moves from the current level to the specified level.*/
+    moveToLevel: Command(0x0, MoveToLevelCommandRequest, 0x0, TlvNoResponse),
+    
+    /** Moves from the current level up or down in a continuous fashion. */
+    move: Command(0x1, MoveCommandRequest, 0x1, TlvNoResponse),
+
+    /** Moves from the current level up or down in a stepwise fashion. */
+    step: Command(0x2, StepCommandRequest, 0x2, TlvNoResponse),
+
+    /** Stops moving the current level. */
+    stop: Command(0x3, StopCommandRequest, 0x3, TlvNoResponse),
+
+    /** Same as {@link commands.moveToLevel}, but change the status of OnOff device on same endpoint. */
+    moveToLevelWithOnOff: Command( 0x4, MoveToLevelCommandRequest, 0x4, TlvNoResponse),
+
+     /** Same as {@link commands.move}, but change the status of OnOff device on same endpoint. */
+    moveWithOnOff: Command(0x5, MoveCommandRequest, 0x5,TlvNoResponse),    
+    
+    /** Same as {@link commands.step}, but change the status of OnOff device on same endpoint. */
+    stepWithOnOff: Command(0x6, StepCommandRequest, 0x6, TlvNoResponse),
+
+    /** Same as {@link commands.stop}, but change the status of OnOff device on same endpoint. */
+    stopWithOnOff: Command(0x7, StopCommandRequest, 0x7, TlvNoResponse),
+
+    /** Changes current frequency to the requested frequency, or to the closest frequency. */
+    moveToClosestFrequency: OptionalCommand(0x8, MoveToClosestFrequencyCommandRequest, 0x8, TlvNoResponse),
```

Comment:
make take this command (which is only for the frequencvy cluster) out here and then add it just to the Freq-clister by stating:

```
commands: {
    ....commands,
   
    moveToClosestFrquency: ...
}
```